code/modules/spells/spell_types/wizard.dm

```diff
@@ -354,4 +354,19 @@
 		var/mob/living/carbon/C = user
 		playsound(C.loc, 'sound/voice/hiss5.ogg', 80, 1, 1)
 		C.spin(6,1)
+	for(var/atom/movable/AM in thrownatoms)
+		if(distfromcaster == 0)
+			if(istype(AM, /mob/living))
+				var/mob/living/M = AM
+				M.Weaken(5)
+				M.adjustBruteLoss(20)
+				M << "<span class='userdanger'>You're slammed into the floor by the [user]'s tail!</span>"
+		else
+			if(istype(AM, /mob/living))
+				var/mob/living/M = AM
+				M.Weaken(4)
+				M.adjustBruteLoss(10)
+				M << "<span class='userdanger'>You're thrown back by the [user]'s tail!</span>"
+			AM.throw_at_fast(throwtarget, ((Clamp((maxthrow - (Clamp(distfromcaster - 2, 0, distfromcaster))), 3, maxthrow))), 1,user)//So stuff gets tossed around at the same time.
+	

```

Comment:
This entire section is bad. You can't copypaste the other proc wholesale and then call ..() anyway, it will activate twice